index.js

```diff
@@ -30,15 +44,35 @@ module.exports = function( region, bucket, key, args, callback ) {
 				image.quality( Math.min( Math.max( Number( args.quality ), 0 ), 100 ) )
 			}
 
-			// resize & crop
+			// crop (assumes crop data from original)
+			if ( args.crop ) {
+				var cropValues = typeof args.crop === 'string' ? args.crop.split( ',' ) : args.crop
```

Comment:
Could we use `px` suffix, to disambiguate from percentages, like photon does in https://developer.wordpress.com/docs/photon/api/#crop?